### PR TITLE
Add messy but functional item for light switch

### PR DESCRIPTION
**Pull request by alf1e**
(idk why it did other shit)

### CODE REVIEW

File: src/main/java/io/github/fripe070/ghosthunting/block/LightswitchBlockEntity.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
 import net.minecraft.util.math.BlockPos;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.Arrays;
```
Comment (Fripe070) [inline src/main/java/io/github/fripe070/ghosthunting/block/LightswitchBlockEntity.java]: Why was an import added if the only other change in the file was deleting two lines?

File: src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java [hunk 2]
```diff
@@ -14,16 +22,38 @@ public LightSwitchItem(Block block, Settings settings) {
...
-        if (target_block == null) {
-            target_block = context.getBlockPos();
-            context.getWorld().setBlockState(target_block, Blocks.GRAY_CONCRETE.getDefaultState());
-
-            actionResult = ActionResult.SUCCESS;
-        } else {
-            actionResult = super.useOnBlock(context);
-            target_block = null;
+        ActionResult actionResult = ActionResult.PASS;
+        if (!context.getWorld().isClient()) {
```
Comment (Fripe070) [inline src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java]: Invert this if-statement and return early to reduce nesting

File: src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java [hunk 3]
```diff
@@ -14,16 +22,38 @@ public LightSwitchItem(Block block, Settings settings) {
...
+                actionResult = super.useOnBlock(context);
+
+                BlockPos clickPos = context.getBlockPos();
+
+                switch (context.getSide()) {
+                    case NORTH -> clickPos = clickPos.north(1);
+                    case SOUTH -> clickPos = clickPos.south(1);
+                    case EAST -> clickPos = clickPos.east(1);
+                    case WEST -> clickPos = clickPos.west(1);
+                }
```
Comment (Fripe070) [inline src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java]: What does this actually do? And why is it lacking any caases for up/down?

**Comment by Fripe070** [comment]
A proper review will come later when I have time to actually test the code, these are only what I notice from just looking at it

File: src/main/java/io/github/fripe070/ghosthunting/block/LightswitchBlockEntity.java (hunk 1 above)
Comment (alf1e) [reply src/main/java/io/github/fripe070/ghosthunting/block/LightswitchBlockEntity.java]: Was a debuting step. Will fix tonight when I optimise imports

File: src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java (hunk 3 above)
Comment (alf1e) [reply src/main/java/io/github/fripe070/ghosthunting/item/LightSwitchItem.java]: Up/down will be added tonight. Fixed NBT being placed on the block the switch is placed on

**Pull request merged**